Java/crypto/src/main/java/com/breadwallet/crypto/Transfer.java

```diff
@@ -81,4 +82,6 @@
     }
 
     TransferState getState();
+
+    ImmutableSet<? extends TransferAttribute> getAttributes ();
```

Comment:
Should be just `Set` instead of ImmutableSet, no? I guess you are trying to tip hand that it cannot be modified but modifying it actually has no bearing. No reason to limit it. Just my 0.02.

Reply:
Fair enough.  I'll update.